main/zoe/guide/contracts/loan.md

```diff
@@ -76,6 +72,8 @@ forever.  When the borrowing occurs, the collateral is moved to a
 special collateral seat to prevent the lender from exiting with
 collateral before the contract ends due to repayment or liquidation.
```

Comment:
The collateral is moved out of the borrower's seat, so I would phrase it as preventing the **borrower** from exiting early and absconding with it.

Reply:
No, I mean that if there was not a collateralSeat, the collateral would be transferred to the lender's seat (the borrower seat is very short-lived, so the borrower cannot really exit much earlier than it is already exited). If the collateral is transferred to the lender's seat, the lender could exit before the contract ends.

Reply:
I rewrote this part and moved it down the borrower's section